.github/drafts/16506/16506_range_slider_draft.md

```diff
@@ -0,0 +1,68 @@
+| **Min Range** | Formatting | The minimum difference between the Range Slider's starting and ending values. _(number)_ | `{{<widget-name>.minRange}}` |
+| **Default Start Value** | Formatting | This sets the initial value for the slider's lesser/left handle, before the user makes any adjustments. This value should be between the slider's **Min Value** and **Max Value**. _(number)_ | `{{<widget-name>.defaultStartValue}}` |
+| **Default End Value** | Formatting | This sets the initial value for the slider's greater/right handle, before the user makes any adjustments. This value should be between the slider's **Min Value** and **Max Value**. _(number)_ | `{{<widget-name>.defaultEndValue}}` |
+| **Marks** | Formatting | An _array of objects_ with keys `value` and `label` which define where reference labels should appear below the widget's slider. These labels can be shown or hidden with the **Show Marks** property toggle. | `{{<widget-name>.marks}}` |
+| **Tooltip Always On** | Formatting | Hovering over the slider with the mouse cursor shows the slider's selected value in a tooltip; enabling this setting forces the tooltip to always be visible, regardless of the cursor's location. _(bool)_ | `{{<widget-name>.tooltipAlwaysOn}}` |
+| **start** | Binding | Represents the beginning value of the user's selected range. _(number)_ | `{{<widget-name>.startValue}}` |
+| **end** | Binding | Represents the ending value of the user's selected range. _(number)_ | `{{<widget-name>.endValue}}` |
+
+These properties allow you to perform formatting changes or bind it to any other widget in queries or JS objects.
+
+### Min Value / Max Value
+These properties represent the lower and upper bounds of values that can be selected with the slider. The **Min Value** determines the value at the far left end of the widget, and the **Max Value** determines the value at the far right end. Negative and/or decimal values are allowed, however the Min Value must always be less than the Max Value.
+
+![](https://youtu.be/VphLW50YPKo)
+
+### Step Size
+The `Step Size` property determines the smallest increment by which the user may adjust the value selected by the slider. Smaller values for this property (can't be smaller than 0.1) result in more fine control of the selection, whereas larger step sizes result in more coarse control.
+
+![](https://youtu.be/mecwJ-D49gU)
+
+### Min Range
+This property represents the minimum difference allowed between the low and high ends of the user's selection. This value can't be less than 0.1.
+
+![](https://youtu.be/C40N4hh3SRA)
+
+### Default Start / End Value
+When your app is loaded for the first time, or when the page is refreshed, the two ends of the Range Slider widget are automatically set to these values.
```

Comment:
We shouldn't say this, because the default value can updated until the user hasn't interacted with the widget. Which means if you change something in an input widget, the default value of slider can be updated unless the user has edited it.